### PR TITLE
Review and add missing col sorting

### PR DESCRIPTION
**Pull request by tjerman**
These are the columns that I thought would make sense to have sortable; thoughts?

### CODE REVIEW

File: compose/module-field.cue [hunk 1]
```diff
@@ -13,8 +13,8 @@ moduleField: schema.#Resource & {
 	struct: {
 		id: schema.IdField
 		namespace_id: { ident: "namespaceID", goType: "uint64", store: false }
-		module_id: { ident: "moduleID", goType: "uint64", storeIdent: "rel_module" }
-		place: { goType: "int" }
+		module_id: { sortable: true, ident: "moduleID", goType: "uint64", storeIdent: "rel_module" }
+		place: { sortable: true, goType: "int" }
 		kind: { goType: "string" }
```
Comment (vicpatel) [inline compose/module-field.cue]: Maybe kind too?

**Review by vicpatel — APPROVED** [review]

File: federation/module-exposed.cue [hunk 2]
```diff
@@ -16,7 +16,7 @@ exposedModule: schema.#Resource & {
 	struct: {
 		id:          schema.IdField
 		handle:      schema.HandleField
-		name: {}
+		name: { sortable: true }
 		node_id: { ident: "nodeID", goType: "uint64", storeIdent: "rel_node" }
 		compose_module_id: { ident: "composeModuleID", goType: "uint64", storeIdent: "rel_compose_module" }
```
Comment (petergrlica) [inline federation/module-exposed.cue:21]: why in `module-mapping.cue` and not here? or the other way around, why add sorting to the fields that are non-user facing and not used internally in sort? I am referring to `node_id`

File: federation/node.cue [hunk 3]
```diff
@@ -13,11 +13,11 @@ node: schema.#Resource & {
 
 	struct: {
 		id:          schema.IdField
-		name: {}
-		shared_node_id: { ident: "sharedNodeID", goType: "uint64" }
+		name: {sortable: true}
+		shared_node_id: { sortable: true, ident: "sharedNodeID", goType: "uint64" }
 		base_url: { goType: "string", ident: "baseURL" }
```
Comment (petergrlica) [inline federation/node.cue]: add

File: system/dal_connection.cue [hunk 4]
```diff
@@ -7,11 +7,11 @@ import (
...
 		id:     schema.IdField
-		name: { goType: "string" }
+		name: { sortable: true, goType: "string" }
 		handle: schema.HandleField
 
 		// omitting isPrimary and replacing with a special type
-		type: { goType: "string" }
+		type: { sortable: true }
 
 		location: { goType: "geolocation.Full" }
```
Comment (petergrlica) [inline system/dal_connection.cue:16]: not sure how this would sort but maybe worth to take a look?

File: system/user.cue [hunk 5]
```diff
@@ -8,10 +8,10 @@ user: schema.#Resource & {
...
 		id:     schema.IdField
 		handle: schema.HandleField
-		email: {unique: true, ignoreCase: true}
+		email: {sortable: true, unique: true, ignoreCase: true}
 		email_confirmed: {goType: "bool"}
-		username: {unique: true, ignoreCase: true}
-		name: {}
+		username: {sortable: true, unique: true, ignoreCase: true}
+		name: {sortable: true}
 		kind: {goType: "types.UserKind"}
```
Comment (petergrlica) [inline system/user.cue]: add?

**Review by petergrlica — COMMENTED** [review]
lgtm, maybe revise the points I expressed or reply

**Pull request merged**